pandas/core/strings.py

```diff
@@ -328,19 +328,44 @@ def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
 
 def str_startswith(arr, pat, na=np.nan):
     """
-    Return boolean Series/``array`` indicating whether each string in the
-    Series/Index starts with passed pattern. Equivalent to
-    :meth:`str.startswith`.
+    Test if the start of each string element matches a pattern.
+
+    Equivalent to :meth:`str.startswith`.
 
     Parameters
     ----------
-    pat : string
-        Character sequence
-    na : bool, default NaN
+    pat : str
+        Character sequence. Regular expressions are not accepted.
+    na : object, default NaN
+        Object shown if element tested is not a string.
 
     Returns
     -------
-    startswith : Series/array of boolean values
+    startswith : Series or array-like of bool
```

Comment:
I was checking the other methods of the `.str.` accessor, and seems like we're a bit inconsistent with the type. In some cases we use `Series/array` or `Series or array-like`, and in some others `Series or Index of whatever`. I think the latter is more specific, as I don't think currently this method can return a numpy array or a Python list.

We can change it at a later point, as some unifying would be nice. But if you want to change it now to `Series or Index of bool` we'd have this right already.

Also, the `startswith : ` here is something that we don't want to use anymore, unless the function/method returns more than one value. So you can leave just the type.